src/core/Provider.ts

```diff
@@ -60,9 +59,14 @@ const stateToPrintable = ([state, atoms]: [State, AnyAtom[]]) =>
 const isDevStore = (store: Store): store is StoreForDevelopment => {
   return store.length > 2
 }
-export const getDevState = (state: State) => ({ ...state }) // shallow copy XXX might be better ways
-export const getDevAtoms = ({ atoms }: { atoms: AnyAtom[] }) => atoms
-export const subscribeDevAtoms = (
+export const getDebugStateAndAtoms = ({
```

Comment:
Instead of `debug*`, maybe call it `snapshot*`, because the atoms and their state is a snapshot?

Reply:
To me, snapshots are immutable. `atoms` are immutable, but `state` is not.